src/Entity/DTO/CurriculumInventoryInstitutionDTO.php

```diff
@@ -5,9 +5,65 @@
 namespace App\Entity\DTO;
 
 use App\Attribute as IA;
+use OpenApi\Attributes as OA;
 
 #[IA\DTO('curriculumInventoryInstitutions')]
 #[IA\ExposeGraphQL]
+#[OA\Schema(
+    title: "CurriculumInventoryInstitution",
+    properties: [
+        new OA\Property(
+            "id",
+            description: "ID",
+            type: "integer"
+        ),
+        new OA\Property(
+            "name",
+            description: "Name",
+            type: "string"
+        ),
+        new OA\Property(
+            "aamcCode",
+            description: "AAMC code",
+            type: "string"
+        ),
+        new OA\Property(
+            "addressStreet",
+            description: "Street address",
+            type: "string"
+        ),
+        new OA\Property(
```

Comment:
duplicate property, throws 
```
User Warning: Multiple @OA\Property() with the same property="addressStreet":
\App\Entity\DTO\CurriculumInventoryExportDTO->createdAt in /srv/app/src/Entity/DTO/CurriculumInventoryExportDTO.php
\App\Entity\DTO\CurriculumInventoryExportDTO->createdAt in /srv/app/src/Entity/DTO/CurriculumInventoryExportDTO.php
``` when trying to load the docs.

Reply:
:heavy_check_mark: